core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -48,6 +57,9 @@
  */
 @UtilityClass
 public class DateTimeFunction {
+  String timeZoneMax = "+14:00";
+  String timeZoneMin = "-13:59";
+  String timeZoneZero = "+00:00";
```

Comment:
Maybe better to move this along with `isValidMySqlTimeZoneId` to some time zone utility class?

Reply:
Similar thought for your reference: https://github.com/opensearch-project/sql/pull/835#discussion_r974804923

Reply:
Fixed in abae9618